scripts/trlite

```diff
@@ -446,7 +446,7 @@ if [ "$RUN" == "all" -o "$RUN" == "4" ]; then
     try_command "ashell" make $VERBOSE ashell ROM=250
 
     # build ide version
-    try_command "ide" make $VERBOSE ide ROM=250
+    try_command "ide" make $VERBOSE ide ROM=251
```

Comment:
probably better make it 256, since the PME will increase to it  anyway,  I tested with the 0.9 SDK and the largest we can do is 259.